### ISSUE TITLE
Username doesn't default to key name in 1.11.1

### ISSUE THREAD

**Issue by pecastro** [passff/passff#482]
### General information. Describe your environment

- Versions
  - Operating system: Linux
  - Browser: FF 89.0.2
  - PassFF: 1.11.1
  - Host app: Latest

Status line output: `[11:58:29] show -> (0) no error message`

---

### Actual behaviour
Username doesn't autofill once you select the possible entry on the popup that appears after clicking the pass icon.
Everything works fine with previous version 1.10.6

### Expected behaviour
Username and Password should be populated in fields.

### Steps to reproduce the behaviour
1. Open site
2. Click pass icon next to username/password field
3. Verify that Username field hasn't been auto-filled with password

**Comment by tuxor1337** [comment]
I can't reproduce this with the login form on github.com. Where did you experience this problem?

**Comment by pecastro** [comment]
Any login form from any site.
Pressing CTRL+Y and trying to Fill using the toolbar context also doesn't seem to work.

**Comment by tuxor1337** [comment]
Would you please try to reproduce this with a clean (new) Firefox profile? I don't experience any problems with the form filling, using the same setup that you describe.

**Comment by Alexey104** [comment]
I can confirm the issue with 1.11.1. When use autofill, only the password field is being filled, but the login field remains empty:
![Arch_login](https://user-images.githubusercontent.com/38672094/125538948-4ff8d080-f52c-449b-a73f-39b41d89faa9.gif)
![Lichess_login](https://user-images.githubusercontent.com/38672094/125538953-7f779623-1aea-45c0-a589-c772b5d09f21.gif)

**Comment by tuxor1337** [comment]
As I said, I can't reproduce this and ask you to please reproduce this with a clean Firefox profile. Alternatively, please post screenshots of your PassFF preferences.

**Comment by Alexey104** [comment]
@tuxor1337, Hello!
I've just completely removed `~/.mozilla` directory, started with new profile, installed `passff` and `hostapp`, and the behavior is the same with clean profile and default `passff` settings(Firefox 90.0). These are the settings I use, but, as I said, with the default settings behavior is the same:
<details>
  <summary>Settings</summary>
      
  ```
  {                                                                                     
  "autoFill": false,
  "autoFillBlacklist": "",
  "autoFillDomainCheck": false,
  "autoSubmit": false,
  "caseInsensitiveSearch": true,
  "contextMenu": true,
  "ctrlEnterBehavior": 3,
  "defaultIncludeSymbols": true,
  "defaultPasswordLength": 16,
  "directoriesFirst": false,
  "enableLogging": false,
  "enterBehavior": 2,
  "filterPathRegex": "notes\nattributes\nattachments",
  "handleHttpAuth": true,
  "indexMetaUrls": false,
  "loginFieldNames": "login,user,username,id",
  "loginInputNames": "login,user,mail,email,tel,username,opt_login,log,usr_name",
  "lookMenuWidth": "28rem",
  "markFillable": false,
  "otpInputNames": "otp,code",
  "otpauthFieldNames": "otpauth,otp",
  "passwordFieldNames": "passwd,password,pass",
  "passwordInputNames": "passwd,password,pass",
  "preferInsert": 0,
  "recognisedSuffixes": "co.uk,org.uk,me.uk,co.jp,com.au",
  "shiftEnterBehavior": 4,
  "showNewPassButton": true,
  "showStatus": true,
  "submitFillable": true,
  "tbMenuShortcut": "Ctrl+Y",
  "urlFieldNames": "url,http"
}
  ```
  
</details>

**Comment by sli720** [comment]
I have the same issue. When I downgrade to version 1.10.6 it works.

**Comment by tuxor1337** [comment]
I tried to reproduce this with both Firefox 89.0.2 and Firefox 90.0, with and without a clean profile and I still can't reproduce this issue in any of the setups I tried, sorry everyone.

**Comment by pecastro** [comment]
@tuxor1337 To clarify the nature of the bug, I'm witnessing the same as what @Alexey104 pasted.
The password field actually does get populated but the username stays empty. ( I've amended the original description of the bug ) 
My version of the pass binary is `v1.7.3` and the hostapp has been installed with the default `curl -sSL github.com/passff/passff-host/releases/latest/download/install_host_app.sh | bash -s -- firefox` 
And just to reitereate

**Comment by tuxor1337** [comment]
I found that you actually don't described a bug in the form-filling, but in the interpretation of pass entries. This has been fixed in 1.11.2 (036cd9e).

**Comment by Alexey104** [comment]
Works great now, thank you!
